site/it/README.md

```diff
@@ -45,19 +57,25 @@ Alcune parole fanno parte di un vocabolario tecnico inglese e pertanto non hanno
 *   maintainer(s)
 *   merge(s)
 *   multicore
+*   namespace(s)
 *   notebook(s)
-*   overfit(ting)
+*   open source
 *   pattern(s)
 *   pixel(s)
-*   pull request(s)
-*   repository(ies)
+*   plug-in(s)
+*   previewer(s)
```

Comment:
Anteprima?

Reply:
Anteprima is preview: the noun.
Previewer (something or somebody who previews) has not a correspondence in Italian...